apps/files_sharing/lib/Cache.php

```diff
@@ -46,27 +47,22 @@
  * don't use this class directly if you need to get metadata, use \OC\Files\Filesystem::getFileInfo instead
  */
 class Cache extends CacheJail {
-	/** @var \OCA\Files_Sharing\SharedStorage */
+	/** @var SharedStorage */
 	private $storage;
-	/** @var ICacheEntry */
-	private $sourceRootInfo;
-	/** @var IUserManager */
-	private $userManager;
-
-	private $rootUnchanged = true;
-
-	private $ownerDisplayName;
-
-	private $numericId;
+	private ICacheEntry $sourceRootInfo;
+	private bool $rootUnchanged = true;
+	private ?string $ownerDisplayName = null;
+	private int $numericId;
```

Comment:
```suggestion
    /** @var int */
	private $numericId;
```

Let's revert this change